software/e-label.yml

```diff
@@ -0,0 +1,17 @@
+name: "E-Label"
+website_url: "https://filipecarneiro.github.io/ELabel/"
+source_code_url: "https://github.com/filipecarneiro/ELabel"
+description: "Open-source solution for electronic labels, with QR Codes, on wine bottles sold within the European Union. This initiative seeks to enhance transparency, efficiency, and information accessibility in the wine industry."
+licenses:
+  - MIT
+platforms:
+  - .NET
+  - C#
+  - Docker
+  - K8S
+tags:
+  - Miscellaneous
+  - Community-Supported Agriculture (CSA)
+  - Inventory Management
+  - URL Shorteners
+demo_url: "https://qr.doutorpc.net/l/GR904-2015"
```

Comment:
> Demo links should only be used for interactive demos

I wouldn't count this static website as interactive, but maybe other reviews want to share their opinion on that.
```suggestion
```